### DISCUSSION TITLE
Using Lua scripts

### DISCUSSION THREAD

**Discussion by Billy60** [FrSkyRC/ETHOS-Feedback-Community#4445] [Q&A]
Good morning.
Where can I find comprehensive info about using Lua scripts on my Horus X10ex ?
There is not much (nothing in fact) on the user manual.
Need to know:
* How come my Tx does not show the item 'lua sources ' in the Model Config menu ?
*  where should I put the lua files on the SDcard ?
*  how do I get the 'run script' item when there is a lua file in the "scripts" folder ?
* where do I find Frsky lua scripts for Rxs such as S8R since it seems to be a old one and the folder where the scripts for Cal and Setup is almost empty ?
* how to connect a S8R to a PC to set it up with the Freelink App on a PC ? I would rather do this with a Lua script !
Thanks

**Comment by wienne** [comment]
我也和你一样在寻求关于脚本有关的资料，
但确实很少。
与你分享我所知的一点点内容，

我的遥控器是Frsky Twin XLite S，
![1](https://github.com/user-attachments/assets/8db62d65-86cb-4b19-b81f-7880458a3190)


安装了一个“devmon” 插件，
可以在主屏幕上加载它，
显示更多的摇杆信息
![screenshot-2024-09-21-61062_1](https://github.com/user-attachments/assets/ea64216a-c807-4c27-928f-2ebff8d72735)
![screenshot-2024-09-21-61043_1](https://github.com/user-attachments/assets/0f3f2e8e-a9f7-4f99-942f-e508e84995d5)

安装的第二是ELRS Configure工具，
在待定外置高频头后，在‘配置’选项里可以调用它。
![screenshot-2024-09-21-61495_1](https://github.com/user-attachments/assets/09e748d3-9363-4e1d-9574-d03f9765f5b3)
![screenshot-2024-09-21-61487_1](https://github.com/user-attachments/assets/2aaf0020-6b0c-4804-97ca-9dd1185c4886)
